drivers/gles3/storage/utilities.cpp

```diff
@@ -327,6 +327,8 @@ void Utilities::update_dirty_resources() {
 }
 
 void Utilities::set_debug_generate_wireframes(bool p_generate) {
+	Config *config = Config::get_singleton();
+	config->generate_wireframes = true;
```

Comment:
```suggestion
	config->generate_wireframes = p_generate;
```

This should use the ``p_generate`` parameter instead of always setting to true. 

``set_debug_generate_wireframes()`` is already called in editor_node to ensure that wireframes are available in the editor